src/components/document/canvas.tsx

```diff
@@ -25,14 +25,15 @@ import { DocumentError } from "./document-error";
 import "./canvas.scss";
 
 interface IProps {
+  content?: DocumentContentModelType;
   context: string;
-  scale?: number;
-  readOnly?: boolean;
   document?: DocumentModelType;
-  content?: DocumentContentModelType;
-  showPlayback?: boolean;
+  idClass?: string; // A class applied that identifies this version of the document, ie top-panel-thumbnail
```

Comment:
Perhaps `contextClass` instead of `idClass`, since it seems like it's representing the context within which it occurs rather than the identify of the document itself.